kword/src/commonTest/kotlin/com/mirego/trikot/kword/extensions/StringToKWordKeyExt.kt.kt

```diff
@@ -0,0 +1,11 @@
+package com.mirego.trikot.kword.extensions
+
+import com.mirego.trikot.kword.KWordKey
+
+val String.toKWordKey: KWordKey
+    get() {
```

Comment:
I would use a data class to get the equality of keys

data class MockKWordKey(override val translationKey: String) : KWordKey